utils/oscap-xccdf.c

```diff
@@ -1041,6 +1046,9 @@ bool getopt_xccdf(int argc, char **argv, struct oscap_action *action)
                 } else {
                     action->f_ovals = NULL;
                 }
+
+		if (action->rule != NULL && action->profile == NULL)
+			return oscap_module_usage(action->module, stderr, "Profile needs to be specified when evaluating a single rule!");
```

Comment:
Well, I've tried the following on Fedora/RHEL7 (without specifying profile) and it returned zero without evaluating anything:
`$ oscap xccdf eval /usr/share/xml/scap/ssg/content/ssg-fedora-ds.xml`
`$ oscap xccdf eval /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml`
How exactly can I trigger this "default" profile evaluation?